ui/pages/send/send-footer/send-footer.test.js

```diff
@@ -42,7 +42,7 @@ describe('SendFooter Component', () => {
 
   it('should match snapshot', () => {
     const { container } = renderWithProvider(
-      <SendFooter {...props} />,
+      <SendFooter {...props} disabled={false} />,
```

Comment:
We should extract these changes to a different PR to cleanup our tests. Good work but we don't want to hide these changes in the git history, or confuse the pr review by trying to figure out why these changes were necessary.